sentences/sv/light_HassLightSet.yaml

```diff
@@ -46,9 +46,41 @@ intents:
           brightness: 1
 
       - sentences:
-          - "(färga | ändra | byt) <name> [till] {color}"
+          - "[<dimra>] [upp] <name> [<ljusintensitet>] till <maximal> [<möjliga>]"
+          - "[<dimra>] [upp] <name> till <maximal> [<möjliga>] [<ljusintensitet>]"
+          - "<name> [upp ] [till] <maximal> [<möjliga>] [<ljusintensitet>]"
+        slots:
+          brightness: 100
+
+      - sentences:
+          - "[<dimra>] [ner] <name> [<ljusintensitet>] till <minimal> [<möjliga>]"
+          - "[<dimra>] [ner] <name> till <minimal> [<möjliga>] [<ljusintensitet>]"
+          - "<name> [upp ] [till] <minimal> [<möjliga>] [<ljusintensitet>]"
+        slots:
+          brightness: 1
+
+      - sentences:
+          - "<maximal> [<möjliga>] <ljusintensitet> <i_på> <area>"
+          - "[<dimra>] [upp] [<alla>] <ljuskällor> till <maximal> [<möjliga>] <i_på> <area>"
+          - "[<dimra>] [upp] [<alla>] <ljuskällor> <i_på> <area> till <maximal> [<möjliga>]"
+          - "[upp] till <maximal> [<möjliga>] <ljusintensitet> <i_på> <area>"
+
+        slots:
+          name: "all"
+          brightness: 100
+
+      - sentences:
+          - "<minimal> [<möjliga>] <ljusintensitet> <i_på> <area>"
+          - "[<dimra>] [ner] [<alla>] <ljuskällor> till <minimal> [<möjliga>] <i_på> <area>"
+          - "[<dimra>] [ner] [<alla>] <ljuskällor> <i_på> <area> till <minimal> [<möjliga>]"
+          - "[ner] till <minimal> [<möjliga>] <ljusintensitet> <i_på> <area>"
+        slots:
+          name: "all"
+          brightness: 1
```

Comment:
This looks like duplication from line 29.
It might be a merge problem?

Reply:
Looks merge problems, it seems like these yaml files with a lot of "- sentences:" can really confuse git.

Reply:
sorry for the delay, been away this week. should I just close this PR and create new ones with only one change in each?

Reply:
I think that is easiest. Pull the latest changes to main to your fork and then create new branches from main where you apply one change :)